packages/cashier/src/containers/cashier/cashier.tsx

```diff
@@ -69,15 +70,17 @@ const Cashier = observer(({ history, location, routes: routes_config }: TCashier
     const { is_payment_agent_visible } = payment_agent;
     const { is_from_derivgo } = common;
     const { is_cashier_visible: is_visible, toggleCashier } = ui;
-    const { is_account_setting_loaded, is_logged_in, is_logging_in } = client;
+    const { is_account_setting_loaded, is_crypto, is_logged_in, is_logging_in } = client;
     const is_account_transfer_visible = useAccountTransferVisible();
     const is_onramp_visible = useOnrampVisible();
     const p2p_notification_count = useP2PNotificationCount();
+    const { data: is_has_p2p_supported_currencies } = useIsHasP2PSupportedCurrencies();
     const {
         data: is_p2p_enabled,
         isSuccess: is_p2p_enabled_success,
         isLoading: is_p2p_enabled_loading,
     } = useIsP2PEnabled();
+    const is_p2p_visible = !is_crypto() && is_p2p_enabled && is_has_p2p_supported_currencies;
```

Comment:
@farhan-nurzi-deriv I think we don't need to make changes in this file 🤔

Reply:
Since i change the logic to use `website_status.p2p_config.disabled`, i need to check if the current account is crypto or not to hide the menu. Because, `website_status.p2p_config.disabled` will only check if client country supports p2p or not. So, even if client switch to crypto account, p2p is still enabled.

Reply:
@farhan-nurzi-deriv Same as I explained [above](https://github.com/binary-com/deriv-app/pull/9116#discussion_r1241458128), P2P tab should be hidden from the cashier and mobile drawer when the active account is not supported by P2P, Only the onboarding card should be visible regardless of the current account if the user has any other account supported by P2P.